python/palindrome_checker.py

```diff
@@ -12,3 +14,17 @@ def palindrome (s):
 s = input()
```

Comment:
Take the input as `Enter a string: `.
and the output should be like `The string is not palindrome`

Reply:
Sure assign it to me , i am thankful to edit this

Reply:
Sure